test/verify/check-machines-disks

```diff
@@ -651,6 +668,49 @@ class TestMachinesDisks(VirtualMachinesCase):
         if m.image in ["debian-testing"]:
             self.allow_journal_messages('.* type=1400 .* apparmor="DENIED" operation="open" profile="libvirt.* name="%s.*' % self.vm_tmpdir)
 
+    def testAddDiskCustomPath(self):
+        b = self.browser
+        m = self.machine
+
+        self.createVm("subVmTest1")
+
+        # Prepare file for Custom File disk type
+        m.execute("touch /var/lib/libvirt/novell.iso")
+
+        self.login_and_go("/machines")
+        b.wait_in_text("body", "Virtual machines")
+
+        b.wait_in_text("#vm-subVmTest1-state", "Running")
+        self.goToVmPage("subVmTest1")
+
+        if "debian" not in m.image and "ubuntu" not in m.image:
+            # Run this test only for shut off VM
```

Comment:
This comment confuses me, you say run this test only for shut off Vms, but then you run it with running Vm, and the is not some xfail error message or anything. What i am missing?

Reply:
Ahh, that comment was meant to be for other test case. This one doesn't need to have a comment. removing